### PR TITLE
Script to launch selenium with php

### PR DESCRIPTION
**Pull request by compojoom**
$selenium = new Selenium(array('browser' => 'chrome', 'insider' => false));
$selenium->run();

### CODE REVIEW

File: Selenium.php [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+		{
+			$driver['path'] = __DIR__ . '/' . $config['MicrosoftEdge']['windowsInsider'];
+		}
+		elseif (isset($config[$browser][$this->getOs()]))
+		{
+			$driver['path'] = __DIR__ . '/' . $config[$browser][$this->getOs()];
+		}
+		else
+		{
+			print('No driver for your browser. Check your browser in acceptance.suite.yml and the webDrivers in codeception.yml');
```
Comment (javigomez) [inline Selenium.php]: 🐙

**Comment by javigomez** [comment]
please update readme too with an example.

Thanks

File: Selenium.php [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+	public function run()
+	{
+		if (!$this->isWindows())
+		{
+			$command = $this->testsPath . "vendor/bin/selenium-server-standalone " . $this->getWebDriver() . $this->seleniumParams . ' >> selenium.log 2>&1 &';
+
+			print('executing: ' . $command);
+			exec($command);
+		}
+		else
```
Comment (yvesh) [inline Selenium.php]: Can you remove that else please? :-)

File: Selenium.php [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+	private function getOs()
+	{
+		$os = php_uname('s');
+
+		if (strpos(strtolower($os), 'windows') !== false)
+		{
+			return 'windows';
+		}
+		// Who have thought that Mac is actually Darwin???
+		elseif (strpos(strtolower($os), 'darwin') !== false)
```
Comment (yvesh) [inline Selenium.php]: ~~else~~if

There is stripos btw instead of strpos, no strtolower then :-) Thanks!

File: config.dist.ini [hunk 4]
```diff
@@ -0,0 +1,16 @@
+[firefox]
+windows=bin\webdrivers\gecko\geckodriver64.exe
```
Comment (810) [inline config.dist.ini]: change to 
`windows=bin\webdrivers\gecko\geckodriver.exe`

so we are support 32bit as default

Comment (810) [reply config.dist.ini]: maybe we can improve more like:
````
switch(PHP_INT_SIZE) {
    case 4:
        echo '32-bit version of PHP';
        break;
    case 8:
        echo '64-bit version of PHP';
        break;
    default:
        echo 'PHP_INT_SIZE is ' . PHP_INT_SIZE;
}
````

Comment (compojoom) [reply config.dist.ini]: I don't think that we should go crazy with detection here. At the end you can always change config.dist.ini to config.ini and set the correct driver for your environment

**Comment by yvesh** [comment]
Thank you @compojoom @810

**Pull request merged**